### PR TITLE
Global announcement

### LINKED ISSUE #63: Global announcement on Login-Page
### Description:
Before a user even logs into Rocket.Chat, there may be information related to the system (e. g. about downtimes, required browser version or similar) which shall be published to the user.
This message shall be configurable by the admin

### Server Setup Information:

* Version of Rocket.Chat Server: 0.58.2

### PR DESCRIPTION
**Pull request by mrsimpson**
Adds a configurable, highlighted message to the Login- and Home-Screen.
Closes #63

global announcement home
global announcement login

A downport to RC core is pending. I'll create a new issue for that on RC

### CODE REVIEW

**Comment by mrsimpson** [comment]
There is no funky "User has read an accepted it and thus hides the message"-behavior implemented. This would be more complex and raise the question about lifetime of the announcement.

File: packages/rocketchat-theme/client/imports/base.css [hunk 1]
```diff
@@ -3898,6 +3898,34 @@ body:not(.is-cordova) {
...
+}
+
+#login-global-announcement {
+	width: 100%;
+	max-width: 520px;
+	margin: auto;
+	box-shadow:
+			0 1px 1px 0 rgba(0, 0, 0, 0.2),
+			0 2px 10px 0 rgba(0, 0, 0, 0.16);
+	border-radius: 4px;
```
Comment (ruKurz) [inline packages/rocketchat-theme/client/imports/base.css]: Is there a constant available for the border-radius?

File: packages/rocketchat-theme/client/imports/base.css [hunk 2]
```diff
@@ -3898,6 +3898,34 @@ body:not(.is-cordova) {
...
 }
 
+.global-announcement {
+	padding: 20px;
+	text-align: center;
+}
+
+#login-global-announcement {
+	width: 100%;
+	max-width: 520px;
```
Comment (ruKurz) [inline packages/rocketchat-theme/client/imports/base.css]: Prevent fixed px.

File: packages/rocketchat-theme/client/imports/base.css [hunk 3]
```diff
@@ -3898,6 +3898,34 @@ body:not(.is-cordova) {
...
+			0 1px 1px 0 rgba(0, 0, 0, 0.2),
+			0 2px 10px 0 rgba(0, 0, 0, 0.16);
+	border-radius: 4px;
+	position: relative;
+	z-index: 1;
+}
+
 #login-card {
 	width: 100%;
 	max-width: 520px;
```
Comment (ruKurz) [inline packages/rocketchat-theme/client/imports/base.css]: avoid sizing in fixed px

**Pull request merged**